packages/editor/src/plugins/image-gallery/editor.tsx

```diff
@@ -1,14 +1,73 @@
+import { editorPlugins } from '@editor/plugin/helpers/editor-plugins'
+import { EditorPluginType } from '@editor/types/editor-plugin-type'
+import { useState } from 'react'
+
 import type { ImageGalleryProps } from '.'
 import { AddImages } from './components/add-images'
+import { SingleImageModal } from './components/single-image-modal'
 import { ImageGalleryToolbar } from './toolbar'
 
+export enum ImageGalleryPluginViewType {
+  INIT = 'INIT', // Initial state, no images
+  ADD_IMAGES = 'ADD_IMAGES', // Image selection screen
+  GALLERY = 'GALLERY', // Image grid
+}
```

Comment:
Thanks for using an enum here! If comments are necessary to explain names, that usually tells me that names aren't good enough. Maybe consider:
```suggestion
enum ImageGalleryPluginView {
  EMPTY = 'EMPTY',
  SELECTING = 'SELECTING',
  GALLERY = 'GALLERY',
}
```
Or similar.

Also, couldn't find it being used anywhere else, so doesn't need to be exported.

Reply:
Done